backend/typescript/services/implementations/EntityServicePg.ts

```diff
@@ -51,14 +62,23 @@ class EntityService implements IEntityService {
 
   async createEntity(entity: EntityRequestDTO): Promise<EntityResponseDTO> {
     let newEntity: PgEntity | null;
+    const fileName = entity.filePath ? uuidv4() : "";
     try {
       newEntity = await PgEntity.create({
         string_field: entity.stringField,
         int_field: entity.intField,
         enum_field: entity.enumField,
         string_array_field: entity.stringArrayField,
         bool_field: entity.boolField,
+        file_name: fileName,
       });
+      if (entity.filePath) {
+        this.storageService.createFile(
+          fileName,
+          entity.filePath,
+          entity.fileContentType,
+        );
+      }
```

Comment:
Let's do the file creation before the entity creation like in the Mongo implementation. It's possible to get a dangling file if entity creation fails, but I think that's more preferable than having a dangling file reference (i.e. file creation fails **after** entity creation succeeds, entity's fileUUID does not point to any file)